### PR TITLE
Changed lenta handler keyword

### PR DESCRIPTION
**Pull request by ozalexo**
Removed unnecessary message about amount of  found items

### CODE REVIEW

File: src/handlers/text.ts [hunk 1]
```diff
@@ -22,10 +22,10 @@ export const diameterTextHandler = (context: tfTypes.ContextMessageUpdate) => {
 }
 
 export const lentaSaleTextHandler = async (context: tfTypes.ContextMessageUpdate) => {
-  const matches = context.message.text.toLowerCase().match(/lenta/)
+  const matches = context.message.text.toLowerCase().match(/lentasale/)
   if (matches) {
     const saleList =  await lenta()
-    context.reply(`Found ${saleList.length} items`)
+    // context.reply(`Found ${saleList.length} items`)
```
Comment (e-maximus) [inline src/handlers/text.ts:28]: Why did you commented this line?

File: src/handlers/text.ts [hunk 2]
```diff
@@ -22,10 +22,10 @@ export const diameterTextHandler = (context: tfTypes.ContextMessageUpdate) => {
 }
 
 export const lentaSaleTextHandler = async (context: tfTypes.ContextMessageUpdate) => {
-  const matches = context.message.text.toLowerCase().match(/lenta/)
+  const matches = context.message.text.toLowerCase().match(/lentasale/)
```
Comment (e-maximus) [inline src/handlers/text.ts:25]: Let's do just `/^lenta$/`

File: src/handlers/text.ts (hunk 1 above)
Comment (ozalexo) [reply src/handlers/text.ts:28]: Becasue I don't care how much seles were found. And delay between this message and actual result is also annoying. IMHO.

File: src/handlers/text.ts (hunk 2 above)
Comment (ozalexo) [reply src/handlers/text.ts:25]: Okay

**Pull request merged**